### PR TITLE
RUNTIME-163 MCS Regions data source

### PR DESCRIPTION
**Pull request by irlndts**

### CODE REVIEW

File: mcs/provider.go [hunk 1]
```diff
@@ -71,6 +72,22 @@ func newConfig(d *schema.ResourceData, terraformVersion string) (configer, error
...
+		config.UserDomainID = os.Getenv("OS_USER_DOMAIN_ID")
+	}
+	if config.Password == "" {
+		config.Password = os.Getenv("OS_PASSWORD")
+	}
+	if config.Username == "" {
+		config.Username = os.Getenv("OS_USERNAME")
+	}
+	if config.Region == "" {
+		config.Region = os.Getenv("OS_Region")
```
Comment (Zatonskikh) [inline mcs/provider.go:88]: should be upper case

File: mcs/provider.go [hunk 2]
```diff
@@ -71,6 +72,22 @@ func newConfig(d *schema.ResourceData, terraformVersion string) (configer, error
...
 	}
 
+	if config.TenantID == "" {
+		config.TenantID = os.Getenv("OS_PROJECT_ID")
+	}
+	if config.UserDomainID == "" {
+		config.UserDomainID = os.Getenv("OS_USER_DOMAIN_ID")
+	}
+	if config.Password == "" {
+		config.Password = os.Getenv("OS_PASSWORD")
```
Comment (Zatonskikh) [inline mcs/provider.go:82]: do we refuse to use env variables without 'OS' prefix?

**Review by zabral — APPROVED** [review]

File: mcs/provider.go (hunk 1 above)
Comment (irlndts) [reply mcs/provider.go]: yes, fixed

File: mcs/provider.go (hunk 2 above)
Comment (irlndts) [reply mcs/provider.go:82]: since mcs use openrc files with OS_ prefix it still has sence

**Review by zabral — APPROVED** [review]

**Pull request merged**